src/Mod/Path/Path/Post/scripts/fangling_post.py

```diff
@@ -0,0 +1,450 @@
+                    # A positioning move following a rapid move indicates a new cut is starting
+                    # print ("new cut") 
+                    if OUTPUT_COMMENTS:
+                         out += linenumber() + " (Torch On)\n"
+                    out += linenumber() + " M07\n"
+                    if OUTPUT_COMMENTS:
+                         out += linenumber() + " (Pierce Delay)\n"
+                    out += linenumber() + " G4 P1\n"
+
+            if adaptiveOp and c.Name in ["G0", "G00"]:
+                if opHorizRapid and opVertRapid:
+                    command = 'G1'
+                else:
+                    outstring.append('(Tool Controller Rapid Values are unset)' + '\n')
+
+            # Filter out vertical moves since FangLing plasma cutters does not support them
+            if command in ["G0", "G00"] and len(c.Parameters) == 1 and 'Z' in c.Parameters:
+                #print(c, c.Name, len(c.Parameters), list(c.Parameters))
+                print('vertical move detected and removed')
+            if command in ["G54"]:
+                #print(c, c.Name, len(c.Parameters), list(c.Parameters))
+                print('Set coordinate system removed')
+            else:
+                if command in ["G0", "G00"] and not 'X' in c.Parameters and not 'Y' in c.Parameters:
+                    print('No horizontal move detected so removed')
+                else:
+                    outstring.append(command)
+
+            # if modal: suppress the command if it is the same as the last one
+            if MODAL is True:
+                if command == lastcommand:
+                    outstring.pop(0)
+
+            if c.Name[0] == '(' and not OUTPUT_COMMENTS: # command is a comment
+                continue
+
+            # Now add the remaining parameters in order
+            for param in params:
+                if param in c.Parameters:
+                    if param == 'F' and (currLocation[param] != c.Parameters[param] or OUTPUT_DOUBLES):
+                        if c.Name not in ["G0", "G00"]:  # mach3_4 doesn't use rapid speeds
+                            speed = Units.Quantity(c.Parameters['F'], FreeCAD.Units.Velocity)
+                            if speed.getValueAs(UNIT_SPEED_FORMAT) > 0.0:
+                                outstring.append(param + format(float(speed.getValueAs(UNIT_SPEED_FORMAT)), precision_string))
+                            else:
+                                continue
+                    elif param == 'T':
+                        outstring.append(param + str(int(c.Parameters['T'])))
+                    elif param == 'H':
+                        outstring.append(param + str(int(c.Parameters['H'])))
+                    elif param == 'D':
+                        outstring.append(param + str(int(c.Parameters['D'])))
+                    elif param == 'S':
+                        outstring.append(param + str(int(c.Parameters['S'])))
```

Comment:
```suggestion
                elif param in ['T', 'H', 'D', 'S']:
                    outstring.append(param + str(int(c.Parameters[param])))
```
Also why `str(int(var))` and not just `str(var)`?